code/game/machinery/computer/camera.dm

```diff
@@ -51,8 +51,15 @@ var/list/obj/machinery/camera/cyborg_cams = list(
 	..()
 
 /obj/machinery/computer/security/attack_ai(var/mob/user)
-	src.add_hiddenprint(user)
-	return attack_hand(user)
+	if(istype(user, /mob/living/silicon/robot) || isMoMMI(user))
```

Comment:
borgs and mommis don't call attack_ai at melee range, they call attack_robot()...

Reply:
I'm dumb